### PR TITLE
[MGDOBR-335] Shell scripts update

### PR DESCRIPTION
**Pull request by radtriste**
Mainly avoid calls to other scripts assuming script is run in a specific folder and some other tweaks

[MGDOBR-335](https://issues.redhat.com/browse/MGDOBR-335)

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

* <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

* <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: dev/bin/manager-run.sh [hunk 1]
```diff
@@ -9,12 +9,14 @@
...
+SCRIPT_DIR_PATH=`dirname "${BASH_SOURCE[0]}"`
 
-instance_json="${credentials_folder}/${MANAGED_KAFKA_INSTANCE_NAME}.json"
-admin_json="${credentials_folder}/${MANAGED_KAFKA_INSTANCE_NAME}-admin.json"
-ops_json="${credentials_folder}/${MANAGED_KAFKA_INSTANCE_NAME}-ops.json"
-mc_json="${credentials_folder}/${MANAGED_KAFKA_INSTANCE_NAME}-mc.json"
+. "${SCRIPT_DIR_PATH}/configure.sh" kafka minikube-started managed-connectors
+
+instance_json="${CREDENTIALS_FOLDER}/${MANAGED_KAFKA_INSTANCE_NAME}.json"
+admin_json="${CREDENTIALS_FOLDER}/${MANAGED_KAFKA_INSTANCE_NAME}-admin.json"
```
Comment (sutaakar) [inline dev/bin/manager-run.sh:17]: Can you use `${ADMIN_SA_NAME}` ... also here?

Comment (radtriste) [reply dev/bin/manager-run.sh]: updated. CREDENTIALS_FILE are now global env var coming from configure script

File: dev/bin/minikube-start.sh [hunk 2]
```diff
@@ -11,26 +11,42 @@
...
+
+minikube_opts=""
 if [ -n "${MINIKUBE_DRIVER}" ]; then
-  minikube_driver_flag="--driver=${MINIKUBE_DRIVER}"
+  minikube_opts="${minikube_opts} --driver=${MINIKUBE_DRIVER}"
+fi
+
+if [ -n "${MINIKUBE_CONTAINER_RUNTIME}" ]; then
+  minikube_opts="${minikube_opts} --container-runtime=${MINIKUBE_CONTAINER_RUNTIME}"
 fi
```
Comment (radtriste) [inline dev/bin/minikube-start.sh:30]: Added CONTAINER_RUNTIME to be able to use `cri-o` on my Fedora

File: dev/README.md [hunk 3]
```diff
@@ -104,6 +104,19 @@ Either export this env variable or add this to `localconfig` file (the latter me
 MINIKUBE_DRIVER=hyperkit
 ```
 
+### Setup podman and cri-o container runtime
+
+With Fedora 34, there is an issue in spotting the coredns pod due to a cgroup issue with docker/podman.
+
+One workaround is to use the `cri-o` container runtime with `podman` driver.
+
+Either export this env variable or add this to `localconfig` file (the latter method is suggested):
```
Comment (radtriste) [inline dev/README.md:113]:
```suggestion
Either export those env variables or add this to `localconfig` file (the latter method is suggested):
```

**Review by sutaakar — APPROVED** [review]

**Comment by manstis** [comment]
@radtriste Unfortunately it looks like https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/340 has broken your PR.

**Review by lucamolteni — APPROVED** [review]

File: build-parent/pom.xml [hunk 4]
```diff
@@ -388,5 +388,18 @@
         <impsort.goal>check</impsort.goal>
       </properties>
     </profile>
+    <profile>
+      <id>quickly</id>
+      <activation>
+        <property>
+          <name>quickly</name>
```
Comment (danielezonca) [inline build-parent/pom.xml:395]: Can you please update scripts/MD to mention `-Dquickly`?
See [usages](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/search?q=%22-dskiptests%22)

**Review by danielezonca — APPROVED** [review]
LGTM, one minor comment

**Comment by radtriste** [comment]
tested ok. Ready to be merged :)

**Pull request merged**